indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,130 @@
+        });
+
+        $('#select-none').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', false).trigger('change');
+        });
+
+        $('.sessions .show-session-slots').on('click', function() {
+            $(this).parent().parent().next('tbody.session-blocks').toggle();
+        });
+
+        $('.sessions .toolbar').on('click', '.disabled', function(e) {
+            e.preventDefault();
+            e.stopPropagation();
+        });
+
+        $('#add-new-session').on('click', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        window.location.reload();
+                    }
+                }
+            });
+        });
+
+        $('.edit-session').on('click', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        window.location.reload();
+                    }
+                }
+            });
+        });
+
+        $('.show-sessions-btn').on('click', function() {
+            $(this).closest('tr').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+        });
+
+        $('#remove-selected-sessions').on('indico:confirmed', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            var sessionIds = $('.sessions input:checkbox:checked').map(function() {
+                return $(this).val();
+            }).get();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
+                error: handleAjaxError,
+                traditional: true,
+                data: {session_id: sessionIds},
+                success: function() {
+                    $('.sessions input:checkbox:checked').closest('tr').remove();
+                }
+            });
+        });
+
+        $('.js-export-sessions').on('click', function(ev) {
+            ev.preventDefault();
+            var $this = $(this);
+            var itemsChecked = $.map($('.sessions input:checkbox:checked'), function(obj) {
+                return $(obj).val();
+            });
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
+                traditional: true,
+                error: handleAjaxError,
+                data: {session_ids: itemsChecked}
+            });
+        });
+
+        $('.session-persons-list').on('click', function(ev) {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title')
+            });
+        });
+
+        $('.sessions .toolbar').dropdown();
```

Comment:
we can't use the `js-dropdown` class (there's code in global.js enabling dropdowns for those elements)